### ISSUE TITLE
Use case 3 distinguish between born and unborn patient

### ISSUE THREAD

**Issue by LilianMinne** [HL7/fetal_records#16]
Use case 3 describes data transfer from one system to another system. The use case includes data of the baby patient from both before and after birth. In this use case it is not clear how the fetus profile is used since the patient is born. It would be nice to demonstrate in this use case the change in the born status of the baby after it is born (and of course other changes like the change in the related person). This also brings us back to the discussion whether the fetus profile should include the born status. I think this is a nice use case to show how the status changes from unborn to born. But still the naming "fetus" is a bit odd in this use case. That's why I suggested earlier that we may want to use another name for this profile, although baby/child would on the other hand imply the patient is already born. I cannot think of a name that would cover both the born and unborn stage of the baby/child.

**Comment by techintrorepo** [comment]
@LilianMinne , I agree, the fetus profile name would be inappropriate to be used for a born patient. At this moment it is unclear how this transition to be modeled. I also believe a different name would be better. Something like IBCM Patient.

**Comment by LilianMinne** [comment]
@techintrorepo IBCM Patient is a possibility, but since our IG also includes examples of the Mother Patient, wouldn't it be better to explicitly state in the naming of the profiles whether it's about the mother or the child? And since we are often referring to the foetus as "the unborn child" would it also be a possibility to call it IBCM Child Patient?
